### ISSUE TITLE
Parse timer duration buckets obtained through reflection

### ISSUE THREAD

**Issue by objectiser** [jaegertracing/jaeger-lib#65]
## Requirement - what kind of business use case are you trying to solve?

PR #63 includes support for defining buckets on Timer and Histogram metric types. The buckets can be statically declared with the metrics and parsed on reflection.

## Problem - what in Jaeger blocks you from solving the requirement?

Buckets for histogram metrics are being parsed, but currently not for timer durations.

## Proposal - what do you suggest to solve the problem or improve the existing situation?

Parse the timer durations.

## Any open questions to address

How should the durations be expressed? For readability it would probably be best as a number + unit (e.g. 1s, 10m)

**Comment by yurishkuro** [comment]
> How should the durations be expressed? For readability it would probably be best as a number + unit (e.g. 1s, 10m)

Yes, Go has a standard way of representing durations as strings.
